sormas-ui/src/test/java/de/symeda/sormas/ui/externalmessage/ExternalMessageMapperTest.java

```diff
@@ -1,38 +1,38 @@
-package de.symeda.sormas.ui.labmessage;
+package de.symeda.sormas.ui.externalmessage;
```

Comment:
If the tested class gets moved (based on a review comment, this test class also needs to be moved.